### DISCUSSION TITLE
Bug: - PatternFly5 "Select" component flicks.

### DISCUSSION THREAD

**Discussion by arashpreet0011** [patternfly/patternfly#6252] [Q&A]
I upgrade from patternFly4 to patternFly5. I used "Select" component with typeahead. When we search a option in search input of select component, then choose a option at that time component flicks. Looks like, component closed while choose a option, then list of data reset to initial stage and component reopen and closed once again. This wasn't in patternFly4. This happens only when we select option using search otherwise working is smooth.

This is common issue of "Select" component, anyone can fix this.

**Comment by nicolethoen** [comment]
@arashpreet0011 I'm sorry i'm just seeing this question now - which version of PF5 are you using? I believe that was addressed in a subsequent 5.1.1 release with [this PR](https://github.com/patternfly/patternfly-react/pull/9339).
